net/raft/raft.go

```diff
@@ -304,7 +304,23 @@ func (sv *Service) Init() error {
 
 	peers := []raft.Peer{{ID: firstNodeID, Context: []byte(sv.laddr)}}
 	raftNode := raft.StartNode(sv.config(firstNodeID), peers)
-	sv.startLocked(firstNodeID, raftNode, walobj)
+
+	sv.id = firstNodeID
+	sv.raftNode = raftNode
+
+	// StartNode appends to the initial log a ConfChangeAddNode entry for
+	// each peer. We can't campaign until these entries are applied,
```

Comment:
Might add a couple words saying that "each peer" here is actually just the current node. (It doesn't have to be, you can add as many peers as you want to start with. But the way we use the package it's always just one.)